bin/apidoc

```diff
@@ -55,6 +55,8 @@ var argv = cmd
 
     .option('--encoding <encoding>', 'Set the encoding of the source code. [utf8].', 'utf8')
 
+    .option('--filter-by <tag-filter:value>', 'Filter doc by tag')
```

Comment:
Please use `documentation` in full ;)

Reply:
@NicolasCARPi I don't understand what you mean by that :)

Reply:
instead of "doc" write "documentation"

Reply:
ah hh oké :)

Reply:
Done